model-optimizer/mo/utils/utils.py

```diff
@@ -133,3 +138,36 @@ def unique_by(xs: list, predicate: Callable) -> list:
     """
     groups = group_by_with_binary_predicate(xs, predicate)
     return [group[0] for group in groups]
+
+
+def send_params_info(argv: argparse.Namespace, cli_parser: argparse.ArgumentParser):
+    """
+    This function sends information about used command line parameters.
+    :param argv: command line parameters.
+    :param cli_parser: command line parameters parser.
+    """
+    t = tm.Telemetry()
+    params_with_paths = ['input_model', 'output_dir', 'caffe_parser_path', 'extensions', 'k', 'output_dir',
+                         'input_checkpoint', 'input_meta_graph', 'input_proto', 'input_symbol', 'mean_file',
+                         'mean_file_offsets', 'pretrained_model_name', 'saved_model_dir', 'tensorboard_logdir',
+                         'tensorflow_custom_layer_libraries', 'tensorflow_custom_operations_config_update',
+                         'tensorflow_object_detection_api_pipeline_config', 'tensorflow_use_custom_operations_config',
+                         'transformations_config']
+    for arg in vars(argv):
+        arg_value = getattr(argv, arg)
+        if arg_value != cli_parser.get_default(arg):
+            if arg in params_with_paths:
+                param_str = arg + ":" + str(1)
```

Comment:
is `str(1)` used to avoid confidential information gathered from paths? If yes, it may sense to make a comment here.

Reply:
Yes, you are right, str(1) used instead of paths as they contain confidential information. Added comment.